homeassistant/components/coronavirus/config_flow.py

```diff
@@ -21,20 +22,21 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_user(
         self, user_input: dict[str, Any] | None = None
-    ) -> dict[str, Any]:
+    ) -> FlowResultDict:
         """Handle the initial step."""
-        errors = {}
+        errors: dict[str, str] = {}
 
         if self._options is None:
             coordinator = await get_coordinator(self.hass)
             if not coordinator.last_update_success:
                 return self.async_abort(reason="cannot_connect")
 
             self._options = {OPTION_WORLDWIDE: "Worldwide"}
-            for case in sorted(
-                coordinator.data.values(), key=lambda case: case.country
-            ):
-                self._options[case.country] = case.country
+            if coordinator.data is not None:
```

Comment:
Should we group this condition with the one on line 31 so that we abort if we don't have data?